packages/types/src/localization.ts

```diff
@@ -518,4 +518,11 @@ type _LocalizationResource = {
     form_password_incorrect: LocalizationValue;
     not_allowed_access: LocalizationValue;
   };
+  relativeDates: {
```

Comment:
Let's rename this to `dates` so we can add more props in the future, if needed, under the same top-level key.

If we want to introduce more structure, we can also do `dates.relativeFormat` but I don't feel strongly about it.

These are my suggestions:

```tsx
  dates: {
     ...
  }
```

or 

```tsx
  dates: {
     relativeFormat: { 
         ....
      }
  }
```


Regarding the actual case names now, do you think we can remove the "at" suffix as we don't really enforce a specific format (eg "last {} at {}") - the user can pass any string they want 🙃